homeassistant/components/sensor/nilu_air_quality.py

```diff
@@ -0,0 +1,162 @@
+"""
+Sensor for checking the air quality around Norway.
+
+Data delivered by luftkvalitet.info and nilu.no.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.nilu_air_quality/
+"""
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import ATTR_ATTRIBUTION, CONF_LATITUDE, CONF_LONGITUDE
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+import homeassistant.util.dt as dt_util
+
+REQUIREMENTS = ['niluclient==0.1.0']
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(minutes=30)
+
+CONF_AREAS = 'areas'
+
+AREAS = [
```

Comment:
Why not use [the data](https://github.com/hfurubotten/niluclient/blob/master/niluclient/__init__.py#L19) from the used module?

Reply:
It was meant to limit the configuration input. As far as I know, this means that I have to import the module globally in the py file since the configuration setup is executed once the sensor is loaded, and that wouldn't work well and as far as I know is discouraged in the HA developers documentation. 
If there is a way to fix this I will gladly implement re-use of the module. In the mean time I have changed the name of the const list to reflect proper use of it in the configuration.